frontend/src/js/controllers/ourteamCtrl.js

```diff
@@ -21,7 +21,7 @@
             onSuccess: function(response) {
                 var status = response.status;
                 var results = response.data;
-                if (status == 200) {
+                if ((status == 200) && results.length) {
```

Comment:
@RishabhJain2018 Try to use status related condition in seperate statements. Use results.length inside the status block.